### PR TITLE
add securityContext to allow running with strict PSP

### PR DESCRIPTION
**Pull request by davidkarlsen**
#### What this PR does / why we need it:
Without a securityContext the pod will fail when running in a namespace with a restrictive (e.g. run as non-root) PSP.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [X] Chart Version bumped
- [X] Variables are documented in the README.md

### CODE REVIEW

**Comment by davidkarlsen** [comment]
@integrii @ihoegen @lolimjake

File: stable/kuberhealthy/templates/deployment.yaml [hunk 1]
```diff
@@ -30,6 +30,8 @@ spec:
       automountServiceAccountToken: true
       containers:
       - image: {{ .Values.image.repository }}:{{ .Values.image.tag }}
+        securityContext:
+        {{- toYaml .Values.securityContext | nindent 10 -}}
```
Comment (integrii) [inline stable/kuberhealthy/templates/deployment.yaml:34]: I like this!

**Review by integrii — APPROVED** [review]

**Comment by integrii** [comment]
Thanks for your contribution here @davidkarlsen!

**Pull request merged**